src/cmd/lint.js

```diff
@@ -15,29 +9,6 @@ const log = createLogger(__filename);
 
 export type LinterOutputType = 'text' | 'json';
```

Comment:
This `LinterOutputType` type is used in the `LinterCreatorParams`, and so it is supposed to be moved into the new `"src/util/linter"` module (also I'm wondering why flow didn't complain about it)